api/app_test.go

```diff
@@ -1738,6 +1742,90 @@ func (s *S) TestUpdateAppWithTagsWithoutPermission(c *check.C) {
 	c.Assert(recorder.Code, check.Equals, http.StatusForbidden)
 }
 
+func (s *S) TestUpdateAppWithAnnotations(c *check.C) {
+	a := app.App{
+		Name:      "myapp",
+		Platform:  "zend",
+		TeamOwner: s.team.Name,
+		Metadata: appTypes.Metadata{
+			Annotations: []appTypes.MetadataItem{{Name: "c", Value: "someData"}},
```

Comment:
@pedrokiefer why we can use simple map[string]string for annotations ?

Reply:
I think that's because the metadata is also stored in mongodb as part of the app. And mongodb support for keys including dots is a mess.

Reply:
Sure! I forgot this problem, in the newer version of MongoDB it is allowed, however, I prefer to stay safe with older versions.